docs/source/resources.mdx

```diff
@@ -0,0 +1,50 @@
+# 📄 Resources, Documentation & Examples
+
+Learn how to use Hugging Face in Google Cloud by reading our blog posts, documentation and examples below.
+
+## Blog posts
+
+- [Hugging Face and Google partner for open AI collaboration](https://huggingface.co/blog/gcp-partnership)
+- [Google Cloud TPUs made available to Hugging Face users](https://huggingface.co/blog/tpu-inference-endpoints-spaces)
+- [Making thousands of open LLMs bloom in the Vertex AI Model Garden](https://huggingface.co/blog/google-cloud-model-garden)
+- [Deploy Meta Llama 3.1 405B on Google Cloud Vertex AI](https://huggingface.co/blog/llama31-on-vertex-ai)
+
+## Documentation
+
+- [Google Cloud Hugging Face Deep Learning Containers](https://cloud.google.com/deep-learning-containers/docs/choosing-container#hugging-face)
+- [Google Cloud public Artifact Registry for DLCs](https://console.cloud.google.com/artifacts/docker/deeplearning-platform-release/us/gcr.io)
+- [Serve Gemma open models using GPUs on GKE with Hugging Face TGI](https://cloud.google.com/kubernetes-engine/docs/tutorials/serve-gemma-gpu-tgi)
+- [Generative AI on Vertex - Use Hugging Face text generation models](https://cloud.google.com/vertex-ai/generative-ai/docs/open-models/use-hugging-face-models)
+
+## Examples
```

Comment:
I think we can remove this part now it's available on the left  side navigation bar?